### PR TITLE
Add trigger inputs branch 

### PR DESCRIPTION
**Pull request by AllaMaevskaya**
For O2 GeneralO2-1849
Trigger information in digitizer output

### CODE REVIEW

File: DataFormats/Detectors/FIT/FT0/include/DataFormatsFT0/Digit.h [hunk 1]
```diff
@@ -114,7 +130,8 @@ struct Digit {
   void setStatusFlag(EEventStatus bit, bool value) { mEventStatus |= (value << bit); };
   bool getStatusFlag(EEventStatus bit) const { return bool(mEventStatus << bit); }
   uint8_t getEventStatusWord() const { return mEventStatus; }
-  ClassDefNV(Digit, 5);
+
+  ClassDefNV(Digit, 6);
```
Comment (shahor02) [inline DataFormats/Detectors/FIT/FT0/include/DataFormatsFT0/Digit.h]: why do you increment the version? The class did not change.

File: Steer/DigitizerWorkflow/src/FT0DigitizerSpec.cxx [hunk 2]
```diff
@@ -101,15 +101,16 @@ class FT0DPLDigitizerTask : public o2::base::BaseDPLDigitizer
...
         }
       }
     }
-    mDigitizer.flush_all(mDigitsBC, mDigitsCh, labels);
+    mDigitizer.flush_all(mDigitsBC, mDigitsCh, mDigitsTrig, labels);
 
     // send out to next stage
     pc.outputs().snapshot(Output{"FT0", "DIGITSBC", 0, Lifetime::Timeframe}, mDigitsBC);
     pc.outputs().snapshot(Output{"FT0", "DIGITSCH", 0, Lifetime::Timeframe}, mDigitsCh);
+    pc.outputs().snapshot(Output{"FT0", "DIGITSTRIGGER", 0, Lifetime::Timeframe}, mDigitsTrig);
```
Comment (shahor02) [inline Steer/DigitizerWorkflow/src/FT0DigitizerSpec.cxx]: I think it is better if all detectors name it "TRIGGERINPUT", i.e. ``pc.outputs().snapshot(Output{"FT0", "TRIGGERINPUT", 0...``

**Review by shahor02 — DISMISSED** [review]
Thanks @AllaMaevskaya ! Just 2 minor comments

**Comment by davidrohr** [comment]
@ktf @pzhristov : After ktf's fix alisw/alidist#2740 most of the errors are gone, but we still have 
```
Error in <TSystem::ExpandFileName>: input: $ROOT_DYN_PATH/libRIO, output: $ROOT_DYN_PATH/libRIO
Error in <TSystem::ExpandFileName>: input: $ROOT_DYN_PATH/libCling, output: $ROOT_DYN_PATH/libCling
```
in this PR and other new ones. Any Idea?

File: Steer/DigitizerWorkflow/src/FT0DigitizerSpec.cxx (hunk 2 above)
Comment (AllaMaevskaya) [reply Steer/DigitizerWorkflow/src/FT0DigitizerSpec.cxx]: OK, thank you. This is what I wanted to ask. I will rename it

**Review by shahor02 — APPROVED** [review]
@lietava this implements the input you need

**Pull request merged**